submissions/tests/test_models.py

```diff
@@ -166,3 +178,56 @@ def test_highest_score_hidden(self):
         highest = ScoreSummary.objects.get(student_item=item).highest
         self.assertEqual(highest.points_earned, 1)
         self.assertEqual(highest.points_possible, 2)
+
+
+class TestTeamSubmission(TestCase):
+    """
+    Test the TeamSubmission class
+    """
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.user = TestTeamSubmission.create_user('user1')
+        super(TestTeamSubmission, cls).setUpTestData()
```

Comment:
Nit: python 3 allows you to simply call `super()` without arguments.

Reply:
What is the process/location of bumping the python version?

Reply:
https://github.com/edx/edx-submissions/blob/master/submissions/__init__.py#L2

Reply:
Apologies. Long day.